UnitsNet.Tests/UnitSystemTests.cs

```diff
@@ -257,6 +257,64 @@ public string ScientificNotationUpperIntervalFormatting(double value)
         }
 
         [Test]
+        public void ShouldUseCorrectMicroSign()
+        {
```

Comment:
Yea, it would have been ideal to tap into the code generation and have this test generated for each unit that uses the Micro symbol. The amount I have to contribute to this is dictated by the return on investment for my client though...so that extra level didn't quite make the cut (yet) unfortunately.